libraries/botbuilder-dialogs-adaptive-runtime/src/index.ts

```diff
@@ -374,12 +375,12 @@ function addComposerConfiguration(configuration: Configuration): void {
         environment = userName;
     }
 
-    configuration.file(path.join(botRoot, 'generated', `luis.settings.${environment}.${luisRegion}.json`));
+    configuration.file(path.join(botRoot, 'generated', `luis.settings.${environment}.${luisRegion}.json`), true);
```

Comment:
is there ever a case where we want this parameter to be false?
why not make it the default or only behavior?

Reply:
The underlying library defaults to the opposite behavior (hence the gnarly splice code). I want this behavior to be opt-in only so we can try to be consistent with the underlying library.